app/models/visitor.rb

```diff
@@ -0,0 +1,18 @@
+class Visitor < ActiveRecord::Base
+  include Person
+
+  belongs_to :visit
+  validates :visit, :rank, presence: true
```

Comment:
Me neither. I thought `position` would be misleading, but don't have any good ideas.